Gemfile.lock

```diff
@@ -400,7 +402,8 @@ GEM
     image_processing (1.12.1)
       mini_magick (>= 4.9.5, < 5)
       ruby-vips (>= 2.0.17, < 3)
-    imgproxy (1.2.0)
+    imgproxy (2.0.0)
```

Comment:
Did you intend to do 2.0? The PR title says 2.1.

Reply:
Good catch! that was a typo.